### PR TITLE
Prefer converting to a Task-returning method vs a 'void' returning method

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/13357
Fixes https://github.com/dotnet/roslyn/issues/13356

### CODE REVIEW

File: src/Features/VisualBasic/Portable/VBFeaturesResources.Designer.vb [hunk 1]
```diff
@@ -1141,7 +1141,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.VBFeaturesResources
                         "ocedure", resourceCulture)
             End Get
         End Property
-
+
```
Comment (Pilchie) [inline src/Features/VisualBasic/Portable/VBFeaturesResources.Designer.vb]: ack!  Whitespace!

**Comment by Pilchie** [comment]
:+1:

File: src/Features/VisualBasic/Portable/VBFeaturesResources.Designer.vb (hunk 1 above)
Comment (CyrusNajmabadi) [inline src/Features/VisualBasic/Portable/VBFeaturesResources.Designer.vb]: WTF.  stupid autogenerated files.

**Comment by CyrusNajmabadi** [comment]
retest windows_debug_unit32_prtest please

**Pull request merged**